.gitignore

```diff
@@ -95,6 +95,7 @@ db.sqlite3
 # Migrations
 migrations/
 authors/apps/authentication/migrations/
+authors/apps/authentication/migrations/0001_initial.py
```

Comment:
This line is unnecessary. Just add an asterisk, in the appropriate position, on the previous line.